plugins/python-build/bin/python-build

```diff
@@ -995,7 +995,13 @@ build_package_activepython() {
 
 anaconda_architecture() {
   case "$(uname -s)" in
-  "Darwin" ) echo "MacOSX-x86_64" ;;
+  "Darwin" )
+    case "$(uname -m)" in
+    "arm64" ) echo "MacOSX-arm64" ;;
+    "i386" | "x86_64" ) echo "MacOSX-x86_64" ;;
+    * ) return 1 ;;
```

Comment:
```suggestion
    * ) echo "MacOSX-x86_64" ;;
```

to preserve the original behavior - it may be unlikely that someone is relying on it, but it's less work for now to not ask the question.